### ISSUE TITLE
Week 1 Discussion

### ISSUE THREAD

**Issue by IMOsbo** [IMOsbo/DATA6550-Reproducibility#1]
All right, just getting the discussion started!

--- 

Do we want to switch from Word to... literally anything else?

- Google Docs - this might be better for change tracking? I don't know, I've never really used it much...
- Overleaf / LaTeX - this looks good! I've used this before a fair amount and it always worked pretty well!
    - The biggest problem is project history is limited to only premium accounts, so I don't think this would work :/ <https://www.overleaf.com/user/subscription/plans?plan=student&period=annual>
    - We could all run LaTeX on our own machines and then check the `.tex` file into the repo
        - This might be easier?
- Quarto <https://quarto.org/>
    - I always really irrationally liked Quarto - it lets you generate your documents from either a Markdown file or a `.ipynb`, which could be nice for our project?

Interested in hearing what y'all think! And if you just want to stay with Word, totally fine with that too!

**Comment by Hrogel9007** [comment]
I would prefer google docs as I'm familiar with it, but I'll leave it up to you guys. Which paper should we try to recreate?

**Comment by richardhoehn** [comment]
I have the premium version - would just need your emails address ... **BUT**; I am fine with using Google Docs (without track-changes). That would work well for me too. I understood Dr. Wallin to say that we only submit the PDF of the report form this point on. Do you guys agree?

**Comment by Kwoods132** [comment]
Google docs works for me as well. I've only used Overleaf once before, so I'm not too familiar with it,

**Comment by richardhoehn** [comment]
I think reading our notes a "soft" vote would be to use Google Docs - Would one of you setup?

My Gmail: **rhoehn@gmail.com** for sharing.

Thanks!

**Comment by Kwoods132** [comment]
My gmail is woods.k273@gmail.com

**Comment by richardhoehn** [comment]
Would you guys want to meet over zoom at 5:30pm today?

**Comment by Hrogel9007** [comment]
My gmail is harogeljr9007@gmail.com

**Comment by Hrogel9007** [comment]
I'm good with the meting being at 5:30 today.

**Comment by Kwoods132** [comment]
That works for me as well.

**Comment by richardhoehn** [comment]
Here is the zoom Link: https://mtsu.zoom.us/j/6153192171?omn=89992630849

**Comment by IMOsbo** [comment]
All right, been looking at some papers and I think I might have some candidates!

---

## GROKKING: GENERALIZATION BEYOND OVERFITTING ON SMALL ALGORITHMIC DATASETS

<https://alphaxiv.org/abs/2201.02177>

- Paper from OpenAI showing that if you overfit a model enough on the training data, it'll eventually converge on the validation set too?

![Image](https://github.com/user-attachments/assets/220a7121-627e-43e7-884c-074f9deaa266)

- Can we _really_ get it to reproduce that validation curve increase again?

> Such experiments can be quickly reproduced on a single GPU, and this makes them convenient testbeds for theories of generalization.

- Not sure if that'll be the case or not?
- Here's the code repo: <https://github.com/openai/grok> (My goodness AI people are bad at naming things)
    - It looks hard enough to maybe not work 100% out of the box, but simple enough we can fix it if it breaks 🙂
 

## Idiosyncrasies in Large Language Models

<https://www.alphaxiv.org/abs/2502.12150>

- Paper from CMU showing that they could train a classifier to tell the difference between text written by different LLMs - with pretty high accuracy?

![Image](https://github.com/user-attachments/assets/0dc766b1-cb56-42ea-be9f-fc3bb41bb3aa)

- I wonder if we'll _really_ get numbers that good...
- They have a code repo: <https://github.com/locuslab/llm-idiosyncrasies> (gotta love the one commit git repo)
    - Not sure if this will be hard to run or not?

**Comment by Hrogel9007** [comment]
I like the Grok paper—seems more manageable since it's lighter computationally. We’ll probably need to tweak some of the 2022 code, but that feels doable. What do you all think? Does this feel like a good direction to everyone?

**Comment by Kwoods132** [comment]
Yeah, I think the Grok paper would be a good one to do.

**Comment by IMOsbo** [comment]
Ok, so I was thinking about this during class... Depending on how long it takes to run their actual code, do we want to try to train a neural net on a smaller dataset and see if we get the same results?

If we could get a simple neural net reproducing the same trends on a dataset like the Titanic dataset, we might actually be able to show the general trend of the paper, even if we can't run the entire thing. This might be better than having nothing?

**Comment by Kwoods132** [comment]
I think that might be worth trying if we're not able to get their actual code to run.
